test/legacy-upgrade/check-from-v0.111.0-kafka-sink.td

```diff
@@ -7,17 +7,14 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
-$ set-from-sql var=expected-compression-implicit-create-sql
-SELECT sql FROM expected_compression_implicit_create_sql
-
 > SHOW CREATE SINK compression_implicit;
-"materialize.public.compression_implicit" ${expected-compression-implicit-create-sql}
+materialize.public.compression_implicit "CREATE SINK materialize.public.compression_implicit IN CLUSTER quickstart FROM materialize.public.kafka_sink_from INTO KAFKA CONNECTION materialize.public.kafka_conn (TOPIC = 'kafka-sink') FORMAT JSON ENVELOPE DEBEZIUM;"
```

Comment:
Also the pretty-printing adds a semicolon where there previously was none. Is this a compatibility concern? I think it would be if people would somehow run SHOW CREATE in their scripts, and then expect output without a trailing semicolon. Not sure why they would do either though.

Reply:
Well, I'd say we should just risk this.

It's probably slightly better to have a semicolon than to not have, because if you want to paste these statements into some SQL shell, then it's slightly easier if they already have a semicolon.

Btw. `mzexplore` might actually run into this problem. For example, its cluster cloning functionality might use `SHOW CREATE` in this way. I'm working on `mzexplore`, so I'll fix these things in follow-up PRs.